spel/minimal-linux.json

```diff
@@ -169,9 +169,9 @@
                 }
             ],
             "max_retries": 20,
-            "name": "minimal-centos-8-hvm",
+            "name": "minimal-centos-8-stream-hvm",
```

Comment:
do we want to add another token-field like this? I worry that people might be parsing/searching using `-` delimeters. Which is to say, instead of `minimal-centos-8-stream-hvm` do `minimal-centos-8stream-hvm`